end-to-end/cypress/e2e/portal/pages/home.cy.js

```diff
@@ -53,5 +53,22 @@ describe("portal-ui", () => {
       // cy.contains('Warning!');
       // cy.contains('Mock Entity');
     });
+
+    it("has working preview pages", () => {
+      cy.visit("/");
+      cy.contains("Resources").click();
+      cy.contains("Cell Type Annotations").click();
+      cy.contains("HuBMAP Data Portal Previews demonstrate functionality");
+      cy.contains("Rahul Satija");
+      // Vitessce loads, pulling data from the network.
+      // Are incidental network requests ok, as long as tests don't depend on them?
```

Comment:
We may be able to add corresponding [intercept](https://docs.cypress.io/api/commands/intercept) commands for `storage.googleapis.com`, `data-2.vitessce.io`, and any other domains Vitessce loads from if these are a concern

Reply:
That's a good point! This isn't an active concern on my end - these two tests are copied over from a file that was deleted as part of a reorganization effort, so this comment is from that original file. I think the page content tests should be expanded and will definitely incorporate more network interceptions during the next round of tests.